controllers/apply/awards-for-all/fields.js

```diff
@@ -428,27 +441,23 @@ module.exports = function fieldsFor({ locale, data = {} }) {
     }
 
     function seniorContactRoleField() {
-        function rolesFor(organisationType) {
+        function rolesFor(organisationType, organisationSubType) {
             const ROLES = {
```

Comment:
Main changes here are to add/remove a few, but also to alphabetise the object to make future editing a bit easier.